src/Modules/Backend/tests/Backend/Actions/UserAddTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\tests\Backend\Actions;
+
+use ForkCMS\Modules\Backend\tests\BackendWebTestCase;
+
+final class UserAddTest extends BackendWebTestCase
+{
+    protected const TEST_URL = '/private/en/backend/user-add';
+
+    public function testPageLoads(): void
+    {
+        self::loginBackendUser();
+        self::assertPageLoadedCorrectly(
+            self::TEST_URL,
+            'Add | users | settings | Fork CMS | Fork CMS',
+            [
+                'Display name',
+                'E-mail',
+                'Super admin (grant access to everything)',
+                'Enable CMS access for this account.',
+                'Short date format',
+            ]
+        );
+        self::assertHasLink('Cancel', '/private/en/backend/user-index');
+    }
+
+    public function testEmptyFormShowsValidationErrors(): void
+    {
+        self::loadPage();
+        self::assertEmptyFormSubmission('user', 3);
+    }
+
+    public function testValidData(): void
+    {
+        self::loadPage();
+
+        self::submitForm(
+            'Add',
+            [
+                'user[user][ec05aaca240e74a0604d93f9e5a7caef][displayName]' => 'Jelmer Prins',
```

Comment:
Where is the string `ec05aaca240e74a0604d93f9e5a7caef` generated? Will it always be the same? How can a tester get this?

Reply:
It comes from here: https://github.com/forkcms/forkcms/blob/b1b0c9226fe8fbcf14792848835a9eb553d022b8/src/Core/Domain/Form/TabsType.php#L17
It's always the same indeed, it's the md5 has of the label of the tab like lbl.Authentication.
I used md5 because if I just use the label I get the following error
`The name "lbl.Authentication" contains illegal characters. Names should start with a letter, digit or underscore and only contain letters, digits, numbers, underscores ("_"), hyphens ("-") and colons (":").`

We could use something different as naming strategy though for the tab part of the form, I'm open for suggestions

Reply:
I would prefer something readable. Maybe use: https://symfony.com/doc/current/components/string.html#slugger?